contracts/eosiolib/transaction.h

```diff
@@ -37,67 +26,114 @@ extern "C" {
     * which is processed immediately after the current message's processing
     * ends such that the success or failure of the parent transaction is
     * dependent on the success of the message. If an inline message fails in
-    * processing then the whole tree of transactions and messages rooted in the
+    * processing then the whole tree of transactions and actions rooted in the
     * block will me marked as failing and none of effects on the database will
     * persist.
     *
-    * Because of this and the parallel nature of transaction application,
-    * inline messages may not affect any `scope` which is not listed in
-    * their parent transaction's `scope`.  They also may not read any `scope`
-    * not listed in either their parent transaction's `scope` or `readScope`.
-    *
-    * Inline messages and Deferred transactions must adhere to the permissions
+    * Inline actions and Deferred transactions must adhere to the permissions
     * available to the parent transaction or, in the future, delegated to the
     * contract account for future use.
     */
 
    /**
     * @defgroup transactioncapi Transaction C API
     * @ingroup transactionapi
-    * @brief Define API for sending transactions
+    * @brief Defines API for sending transactions
     *
     * @{
     */
-   void send_deferred(const uint128_t& sender_id, account_name payer, const char *serialized_transaction, size_t size, uint32_t replace_existing = 0);
 
-   /**
-    * cancel deferred transaction
-    * @return 1 if transaction was canceled, 0 if transaction was not found
-    */
+    /**
+     *  Sends a deferred transaction.
+     *
+     *  @brief Sends a deferred transaction.
+     *  @param sender_id - ID of sender
+     *  @param payer - Account paying for RAM
+     *  @param serialized_transaction - Pointer of serialized transaction to be deferred
+     *  @param size - Size to reserve
+     */
```

Comment:
missing documentation for `replace_existing` if this is `0` then if the provided `sender_id` is already in use by an in-flight transaction from this contract, this will turn into a failing assert.  if it is `1` then this transaction will atomically _replace_ the inflight transaction (effectively cancelling it in the process)